enclave_manager/avalon_enclave_manager/enclave_manager.py

```diff
@@ -330,6 +323,73 @@ def create_enclave_signup_data():
     return enclave_signup_data
 
 
+# -----------------------------------------------------------------
+async def wo_process(lock, config, kv_helper, enclave_manager):
+    try:
+        sleep_interval = int(config["EnclaveManager"]["sleep_interval"])
+    except Exception as err:
+        logger.error("Failed to get sleep interval from config file. " +
+                     "Setting sleep interval to 10 seconds: %s", str(err))
+        sleep_interval = 10
+    while True:
+        await lock.acquire()
```

Comment:
Please add a comment why we need a lock and why we need to synchronize between key refresh and work order processing if its required.

Reply:
I will add comment to explain the need to synchronization using lock